vhdbuilder/prefetch/pkg/cni/cni-prefetch.sh.gtpl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -eux
```

Comment:
if someone specifies a binary that doesn't exist in the image, will CI fail on the PR? If not, is there a way we could make that happen? (maybe run the prefetch script as part of CI and verify that it exits with status zero?)

Reply:
yep, Alison is working on a separate PR for the driver script which will include running the script itself as a part of the content tests - might just consolidate tests into that PR

Reply:
went ahead and added a base-level execution test